src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksNewVendorConversionServiceImpl.java

```diff
@@ -289,92 +291,143 @@ protected void setPOAddressOnPaymentWorksVendor(VendorDetail vendorDetail, Payme
+		paymentWorksNewVendor.setRequestingCompanyTaxCountry(paymentWorksNewVendorDetailDTO.getRequesting_company().getTax_country());
+		paymentWorksNewVendor.setRequestingCompanyW8W9(paymentWorksNewVendorDetailDTO.getRequesting_company().getW8_w9());
+		paymentWorksNewVendor.setRequestingCompanyTelephone(paymentWorksNewVendorDetailDTO.getRequesting_company().getTelephone());
+		paymentWorksNewVendor.setRequestingCompanyDuns(paymentWorksNewVendorDetailDTO.getRequesting_company().getDuns());
+		paymentWorksNewVendor.setRequestingCompanyCorporateEmail(paymentWorksNewVendorDetailDTO.getRequesting_company().getCorporate_email());
+		paymentWorksNewVendor.setRequestingCompanyTaxClassificationName(paymentWorksNewVendorDetailDTO.getRequesting_company().getTax_classification().getName());
+		paymentWorksNewVendor.setRequestingCompanyTaxClassificationCode(paymentWorksNewVendorDetailDTO.getRequesting_company().getTax_classification().getCode());
+	}
 
-		extractCustomFields(paymentWorksNewVendorDetailDTO, paymentWorksNewVendor);
-		
-		
-		// remittance address
-		if (ObjectUtils.isNotNull(paymentWorksNewVendorDetailDTO.getRequesting_company().getRemittance_addresses())
-				&& ObjectUtils.isNotNull(paymentWorksNewVendorDetailDTO.getRequesting_company()
-						.getRemittance_addresses().getRemittance_address())
-				&& paymentWorksNewVendorDetailDTO.getRequesting_company().getRemittance_addresses()
-						.getRemittance_address().size() > 0) {
-
-			// should only be 1 address
-			PaymentWorksRemittanceAddressDTO remittanceAddress = paymentWorksNewVendorDetailDTO.getRequesting_company()
-					.getRemittance_addresses().getRemittance_address().get(0);
-
-			paymentWorksNewVendor.setRemittanceAddressId(remittanceAddress.getId());
-			paymentWorksNewVendor.setRemittanceAddressName(
-					getPaymentWorksUtilityService().trimFieldToMax(remittanceAddress.getName(), "remittanceAddressName"));
-			paymentWorksNewVendor.setRemittanceAddressStreet1(
-					getPaymentWorksUtilityService().trimFieldToMax(remittanceAddress.getStreet1(), "remittanceAddressStreet1"));
-			paymentWorksNewVendor.setRemittanceAddressStreet2(
-					getPaymentWorksUtilityService().trimFieldToMax(remittanceAddress.getStreet2(), "remittanceAddressStreet2"));
-			paymentWorksNewVendor.setRemittanceAddressCity(
-					getPaymentWorksUtilityService().trimFieldToMax(remittanceAddress.getCity(), "remittanceAddressCity"));
-			paymentWorksNewVendor.setRemittanceAddressState(remittanceAddress.getState());
-			paymentWorksNewVendor.setRemittanceAddressCountry(remittanceAddress.getCountry());
-			paymentWorksNewVendor.setRemittanceAddressZipCode(remittanceAddress.getZipcode());
-		}
+	private void populateNewVendorRemittanceAddressAttributes(PaymentWorksVendor paymentWorksNewVendor, PaymentWorksNewVendorDetailDTO paymentWorksNewVendorDetailDTO) {
+        if (singleRemittanceAddressExists(paymentWorksNewVendorDetailDTO)) {
```

Comment:
Tabs were not converted to spaces before a number of the PaymentWorks files were originally committed to our repository.  This issue is going to keep reoccurring because of that oversight.
Asking both reviewers: What are your thoughts of ignoring the formatting issues in this review.  Then, having the next user story I work on be to run the Eclipse formatter on all the PaymentWorks files to convert tabs to be 4 spaces to permanently get rid of this issue. No other code changes would be part of that user story.  Thoughts?

Reply:
I agree, great idea

Reply:
Fix PMW formatting user story JIRA created. Thanks for agreeing to this suggestion.